datasets/de/abgeordnetenwatch/crawler.py

```diff
@@ -0,0 +1,125 @@
+                num_batches = math.ceil(batch_total_results / 100)
+
+        for mandate in api_response.get("data"):
+            politician = mandate.pop("politician")
+
+            # Get state from parliament info
+            parliament_info = mandate.pop("parliament_period").pop("label").split(" ")
+            state = parliament_info[0]
+
+            # Create position
+            position = h.make_position(
+                context,
+                name="Member of the {} Parliament".format(state),
+                country="Germany",
+                subnational_area=state,
+            )
+
+            person = context.make("Person")
+            person.id = context.make_id(politician.pop("id"))
+            politician_fullname = politician.pop("label")
+
+            # Get the politician birthday date
+            context.log.info("Get Politician {} birth year".format(politician_fullname))
+            politician_data = crawl_politician(context, politician.pop("api_url"))
+            birthYear = politician_data.pop("year_of_birth")
+            person.add("birthDate", birthYear)
+
+            # Apply politician fullname to Person entity
+            h.apply_name(person, full=politician_fullname)
+
+            # Create occupancy
+            occupancy = h.make_occupancy(
+                context,
+                person,
+                position,
+                True,
+                start_date=mandate.pop("start_date"),
+                end_date=mandate.pop("end_date"),
+            )
+            if occupancy:
+                context.emit(person, target=True)
+                context.emit(position)
+                context.emit(occupancy)
+
+            # Audit mandate
+            context.audit_data(
+                mandate,
+                ignore=[
+                    "id",
+                    "entity_type",
+                    "api_url",
+                    "label",
+                    "id_external_administration",
+                    "id_external_administration_description",
+                    "type",
+                    "info",
+                    "electoral_data",
+                    "fraction_membership",
+                ],
+            )
+
+            # Audit
+            context.audit_data(
+                politician_data,
+                ignore=[
+                    "id",
+                    "entity_type",
+                    "label",
+                    "api_url",
+                    "abgeordnetenwatch_url",
+                    "first_name",
+                    "last_name",
+                    "birth_name",
+                    "sex",
+                    "party",
+                    "party_past",
+                    "education",
+                    "residence",
+                    "occupation",
+                    "statistic_questions",
+                    "statistic_questions_answered",
+                    "ext_id_bundestagsverwaltung",
+                    "qid_wikidata",
```

Comment:
ah wikidata QID is a high value field for us - if you put that in the `wikidataId` property, we won't need to de-duplicate entities that already exist in the system with such an ID.

Reply:
Done in my last push